conf/uppmax.config

```diff
@@ -17,9 +17,36 @@ process {
 params {
   saveReference = true
 
-  max_memory = 128.GB
+  max_memory = 125.GB
   max_cpus = 16
   max_time = 240.h
   // illumina iGenomes reference file paths on UPPMAX
   igenomes_base = '/sw/data/uppnex/igenomes/'
 }
+
+def hostname = "hostname".execute().text.trim()
+
+if (hostname ==~ "b.*") {
+  params.max_memory = 109.GB
+}
+
+if (hostname ==~ "i.*") {
+  params.max_memory = 250.GB
+}
+
+if (hostname ==~ "r.*") {
+  params.max_cpus = 20
+}
+
+profiles {
+  devel {
+    params {
+      config_profile_description = 'Testing & development profile for UPPMAX, provided by nf-core/configs.'
+      // Max resources to be requested by a devel job
+      max_memory = 120.GB
+      max_time = 1.h
+    }
+    executor.queueSize = 1
+    process.queue = 'devel'
```

Comment:
Is the devel queue available on all clusters?

Reply:
We had the `uppmax_devel` profile before, so I'm guessing the one responsible for that did check it out ;-)